index.html

```diff
@@ -265,8 +288,15 @@ <h3 id="other-deliverables"> Other Deliverables </h3>
             Recommendation, and which must be completed with an implementation
             report before transition to Proposed Recommendation. Additional
             tests may be added to the test suite at any stage of the
-            Recommendation track, and the maintenance of a implementation report
-            is encouraged. </dd>
+            Recommendation track, and the maintenance of an implementation
+            report is encouraged. </dd>
+          <dd> Additionally, this Working Group will seek to define an testing
+            API surface for the Presentation API and for the Remote Playback API
+            with a view to improve test automation, as well as to allow
+            developers to test their own presentations. Depending on progress,
```

Comment:
Consider being more explicit, e.g.:

s/own presentations/own web applications using the above-mentioned APIs/